app/scripts/components/Artur/userLoginArtur/userLoginArtur.component.js

```diff
@@ -0,0 +1,23 @@
+angular.module('auth')
+    .component('userLogin', {
+        templateUrl: '/scripts/components/Artur/userLoginArtur/userLoginArtur.template.html',
+        controller: function ($scope, $location, authService) {
+            this.userName = null;
+
+            this.redirect = function () {
+                $location.path('/home');
+                $('#exampleModal').modal('hide');
+            };
+
+            this.login = function () {
+                if (this.userName) {
+                    authService.login(this.userName);
+                }
```

Comment:
Could you please format the code here (keep "else" after closed "if" block, like this, `} else {` )